q2_moshpit/eggnog/tests/test_dbs.py

```diff
@@ -117,3 +124,131 @@ def test_fetch_diamond_db(self, subp_run):
 
         # Check that commands are ran as expected
         subp_run.assert_has_calls([first_call, second_call], any_order=False)
+
+    @patch("subprocess.run")
+    def test_fetch_eggnog_fasta(self, subp_run):
+        # Call function. Patching will make sure nothing is
+        # actually ran
+        eggnog_fa = fetch_eggnog_proteins()
+        fasta_file = os.path.join(str(eggnog_fa), "e5.proteomes.faa")
+        taxonomy_file = os.path.join(str(eggnog_fa), "e5.taxid_info.tsv")
+
+        # Check that command was called in the expected way
+        first_call = call(
+            [
+                "wget", "-e", "robots=off", "-O", f"{fasta_file}",
+                "http://eggnog5.embl.de/download/eggnog_5.0/e5.proteomes.faa"
+            ],
+            check=True
+        )
+        second_call = call(
+            [
+                "wget", "-e", "robots=off", "-O", f"{taxonomy_file}",
+                "http://eggnog5.embl.de/download/eggnog_5.0/e5.taxid_info.tsv"
+            ],
+            check=True,
+        )
+
+        # Check that commands are ran as expected
+        subp_run.assert_has_calls([first_call, second_call], any_order=False)
+
+    @patch("q2_moshpit.eggnog._dbs._write_version_tsv")
+    @patch("subprocess.run")
+    def test_fetch_ncbi_taxonomy(self, subp_run, w_v_tsv):
+        # Call function. Patching will make sure nothing is actually ran
+        ncbi_data = fetch_ncbi_taxonomy()
+        zip_path = os.path.join(str(ncbi_data), "taxdmp.zip")
+        nodes_path = os.path.join(str(ncbi_data), "nodes.dmp")
+        names_path = os.path.join(str(ncbi_data), "names.dmp")
+        proteins_path = os.path.join(str(ncbi_data), "prot.accession2taxid.gz")
+        version_path = os.path.join(str(ncbi_data), "version.tsv")
```

Comment:
```suggestion
file_names = ["taxdmp.zip", "nodes.dmp", "names.dmp", "prot.accession2taxid.gz", "version.tsv"]
file_paths = [os.path.join(str(ncbi_data), file_name) for file_name in file_names]
zip_path, nodes_path, names_path, proteins_path, version_path = file_paths
```

you could add the same loop here.